ports/espressif/boards/flipperzero_wifi_dev/mpconfigboard.mk

```diff
@@ -0,0 +1,16 @@
+USB_VID = 0x303A
+USB_PID = 0x4001
```

Comment:
You mentioned that this is the existing PID: do you mean on the Flipper Zero or on the WiFi module only?

Reply:
When the wifi dev board has its stock firmware, thats where I got the vid,pid

Reply:
got it, thanks

Reply:
Looks like this is the IDF default PID: https://github.com/espressif/idf-extra-components/blob/master/usb/esp_tinyusb/usb_descriptors.c#L11-L21

I wouldn't use it though because CircuitPython presents different interfaces than what 0x4001 is for. Please request a custom PID from Espressif: https://github.com/espressif/usb-pids